src/main/openapi/location/v1/location-v1.yaml

```diff
@@ -20,10 +20,21 @@ components:
       type: integer
       minimum: 10000
       maximum: 99999
+    BelgianPostCode:
+      description: >
+        Post code (a.k.a postal code, zip code etc.) of a location in Belgium.
```

Comment:
My proposition is to just keep the first part:

`Post code (a.k.a postal code, zip code etc.) of a location in Belgium.`

The rest isn't commonly necessary to understand the field, having it only in fedvoc excel should suffice.
We could add the reference to list of post codes to fedvoc excel in comments column: 

A [list of Belgian post codes](https://www.bpost.be/fr/outil-de-validation-de-codes-postaux) is published by bpost, and is also available as [open dataset](https://data.europa.eu/data/datasets/https-www-odwb-be-explore-dataset-postal-codes-belgium-).

This is modified a bit bc of liberalization of Belgian post market.

will need to review this in functional WG